src/pages/exam.tsx

```diff
@@ -141,13 +162,25 @@ const Exam = () => {
               </li>
             ))}
           </div>
-          <button type="submit">Siguiente</button>
+          <div>
+            {currentQuestionIndex > 0 && (
+              <button onClick={handlePreviousQuestion} type="submit">
+                Anterior
+              </button>
+            )}
+            {questions.length > 0 && currentQuestionIndex < questions.length - 1 && (
+              <button onClick={handleNextQuestion} type="submit">
+                Siguiente
+              </button>
+            )}
+            {currentQuestionIndex === questions.length - 1 && (
+              <button onClick={surveyEnd} type="button">
+                Finalizar
+              </button>
+            )}
+          </div>
         </form>
-      ) : (
-        <button onClick={surveyEnd} type="button">
-          Finalizar
-        </button>
-      )}
+      ) : null}
```

Comment:
Maybe this null in not needed if you use short-circuit operator (&&)

Reply:
Done